File: CodeFestival/Q31-Q40/Q38_호준이의_아르바이트.js
let scores = prompt("점수를 입력하세요.").split(' ').map(function(n) {
    return parseInt(n, 10);
}); // 생각도 못한 방법 바로 이렇게 int로 받아올수있다!

scores.sort((a, b) => {
    return a-b;
}); // 오름차순 정렬

let count = 0;
let arr = [];

while (arr.length < 3) {
    let n = scores.pop();
    if (!arr.includes(n)) {
        arr.push(n);
    }
    count += 1;
}

console.log(count);
// 원래 배열에서 pop해서 마지막 원소 얻어오고, 없으면 넣어주고, 있으면 배열에 안넣지만 count를 +1 해주면서 몇개인지까지 세어주기.
// 내가 생각했던 방법은 파이썬에서 간단하게 적용이 가능하지만, js에서는 힘들다. 이러한 다른 방법들도 생각해보아야한다.

File: CodeFestival/Q31-Q40/Q38_호준이의_아르바이트.py
scores = list(map(int, input().split(" ")))
dupli = list(set(scores))
dupli.sort(reverse=True)
dupli = dupli[0:3]

c = scores.count(dupli[0]) + scores.count(dupli[1]) + scores.count(dupli[2])
print(c)

# user_input = input()

# l = list(user_input.strip().split())
# l = [int (i) for i in l]

# count = 3
# #3개는 무조건 구매, 배열 정렬 후 1~3위 중 중복되는 숫자까지 포함

# data_sorted = sorted(list(l))

# print(data_sorted)
# for i in range(len(l)-1, 0, -1):
# 	if data_sorted[-3] == l[i]:
# 		count += 1
# print(count)